frontend/ui/src/services/image/index.ts

```diff
@@ -0,0 +1,6 @@
+import _, {SyntheticEvent} from 'react';
+
+export const fallbackImage = (event: SyntheticEvent<HTMLImageElement, Event>, channelSource: string) => {
+  event.currentTarget.src = `https://s3.amazonaws.com/assets.airy.co/${channelSource}_avatar.svg`;
+  event.currentTarget.alt = 'fallback image';
```

Comment:
`default ${source} image` maybe?

Reply:
as we use the _avatar name exactly for those images, I would keep it.